.circleci/config.yml

```diff
@@ -12,6 +12,10 @@ jobs:
     machine: true
     steps:
       - checkout
+      - persist_to_workspace:
+          root: ~/
+          paths:
+            - project
```

Comment:
Circle CI was broken, this part was removed incorrectly https://github.com/auth0-samples/auth0-angular-samples/commit/79e2ab72bd642242f1791aed6a251bbc1a1b769b

Reply:
the "checkout" job is not doing anything other than `checkout` (cloning the GH PR). I wonder if this job can be removed, and this command put on top of the `sample-01` job. I assume this was done this way in case new samples were added later, which never happened

Reply:
Yep, I considered that as well. For now I went with revering what was incorrectly removed in the first place instead of changing the structure of things.